Gemfile

```diff
@@ -26,7 +27,3 @@ group :test do
   gem 'simplecov', require: false
   gem 'site_prism'
 end
-
-group :build do
```

Comment:
what are your thoughts on locking it to `:build` and `:development` rather than as a global gem?